Header header
#batch1
uint16 time
float32 accelerationX
float32 accelerationY
float32 accelerationZ
float32 angularRateX
float32 angularRateY
float32 angularRateZ
uint8 navStatus
#bath2
float64 latitude
float64 longitude
float32 altitude
float32 velocityNorth
float32 velocityEast
float32 velocityDown
float32 heading
float32 pitch
float32 roll
